res/controllers/common-hid-packet-parser.js

```diff
@@ -1252,6 +1252,7 @@ class HIDController {
             "[Sampler6]",
             "[Sampler7]",
             "[Sampler8]",
+            "[Main]",
```

Comment:
what group is this again? we also need to add `[App]` IIRC.

Reply:
There seem to have been recent changes that moved, say, vu_meter_l to the "[Main]" group. This list needs to be kept up to date or HID controllers stop working.

Reply:
Added [App]